models/droidMetadata.go

```diff
@@ -14,6 +14,7 @@ type DroidMetadata struct {
 	Product      string                `yaml:"product"`
 	Storage      bool                  `yaml:"storage"`
 	Environments []DroidMetadataEnvDef `yaml:"environments"`
+	Receivers    []string              `yaml:"receivers"`
```

Comment:
I like this, but change the name from receivers to something else. When reading the metadata.yml out of context, it is not clear to the user that this for reporting.

Reply:
I would say `owners` is a better term. And it also helps people to identify the actual owner of the test images.

Reply:
Change it to owner, and this is good to go.

Reply:
Fixed